CMakeLists.txt

```diff
@@ -92,11 +105,11 @@ if(NOT DAEMON_EXTERNAL_APP)
     elseif(WIN32)
         set(USE_CURSES ON)
     else()
-	option(USE_CURSES "Compile with curses support" 1)
+        option(USE_CURSES "Enable fancy colors in terminal's output (does not detect if output is a file)" ON)
```

Comment:
Curses is not used unless you pass the `-curses` flag at runtime. So it's not good to imply that just having this option on could break something with the output.

Reply:
Alright.